openstf_dbc/services/prediction_job.py

```diff
@@ -49,6 +82,7 @@ def get_prediction_job(self, pid, model_type=None, is_active=None):
         # Add model group
         prediction_job = self._add_model_type_group_to_prediction_job(prediction_job)
 
+        prediction_job = self._create_prediction_job_object(prediction_job)
```

Comment:
minor suggestion:
```
        # Add model group
        prediction_job_dict = self._add_model_type_group_to_prediction_job(prediction_job_dict)

        prediction_job = self._create_prediction_job_object(prediction_job_dict)
```
(Also change prediction_job to prediction_job_dict on first assignment)